src/ng-datepicker/ng-datepicker.component.sass

```diff
@@ -9,7 +9,7 @@ $color: #1A91EB
 
 .ngx-datepicker-container
   position: relative
-
+  z-index: 999;
```

Comment:
I think 99 would be enough. :)

Reply:
@jkuri, I would like to avoid big `z-index` values. This may break compatibility with some libraries with modal dialogs (like Bootstrap).